redash/utils/sql_query.py

```diff
@@ -0,0 +1,45 @@
+import sqlparse
+from redash.utils import mustache_render
+from cached_property import cached_property
+
+
+class SQLQuery(object):
+    def __init__(self, template):
+        self.template = template
+        self.query = template
+
+    def apply(self, parameters):
+        if self.__dict__.has_key('text'):
```

Comment:
```suggestion
        if 'text' in self.__dict__:
```

Reply:
Already did this on a local commit and haven't pushed yet, so I'm politely ignoring.